src/BloomExe/Edit/EditingView.cs

```diff
@@ -711,6 +717,7 @@ private void OnChangeImage(DomEventArgs ge)
 				try
 				{
 					imageInfo = PalasoImage.FromFile(existingImagePath);
+					_originalSize = imageInfo.Image.PhysicalDimension;
```

Comment:
OK, so how about if I change... the copyright? What if a future tool lets me change the contrast, etc? I.e. it's not going to work to try and detect that nothing change, short of a byte comparison. I suppose the reason you don't just replace the file is something about replacing the wrong file?